### ISSUE TITLE
launch_requirements.txt version mismatches

### ISSUE THREAD

**Issue by erinn** [onnela-lab/beiwe-backend#56]
It looks like your launch_requirements.txt file could use an update as this conflict currently exists:
ERROR: boto3 1.9.152 has requirement botocore<1.13.0,>=1.12.152, but you'll have botocore 1.7.18 which is incompatible.
ERROR: s3transfer 0.2.0 has requirement botocore<2.0.0,>=1.12.36, but you'll have botocore 1.7.18 which is incompatible.

This in turn MAY be leading towards a failure to bring up an environment:
  File "launch_script.py", line 572, in <module>
    do_create_environment()
  File "launch_script.py", line 349, in do_create_environment
    create_finalized_configuration(name)
  File "/home/user/repo/beiwe-backend/cluster_management/deployment_helpers/configuration_utils.py", line 211, in create_finalized_configuration
    config["S3_BUCKET"] = create_data_bucket(eb_environment_name)
  File "/home/user/repo/beiwe-backend/cluster_management/deployment_helpers/aws/s3.py", line 26, in create_data_bucket
    if check_bucket_name_available(name):
  File "/home/user/repo/beiwe-backend/cluster_management/deployment_helpers/aws/s3.py", line 16, in check_bucket_name_available
    s3_resource = create_s3_resource()
  File "/home/user/repo/beiwe-backend/cluster_management/deployment_helpers/aws/boto_helpers.py", line 60, in create_s3_resource
    region_name=GLOBAL_CONFIGURATION["AWS_REGION"],
  File "/home/user/.local/lib/python2.7/site-packages/boto3/__init__.py", line 100, in resource
    return _get_default_session().resource(*args, **kwargs)
  File "/home/user/.local/lib/python2.7/site-packages/boto3/session.py", line 389, in resource
    aws_session_token=aws_session_token, config=config)
  File "/home/user/.local/lib/python2.7/site-packages/boto3/session.py", line 263, in client
    aws_session_token=aws_session_token, config=config)
  File "/home/user/.local/lib/python2.7/site-packages/botocore/session.py", line 861, in create_client
    client_config=config, api_version=api_version)
  File "/home/user/.local/lib/python2.7/site-packages/botocore/client.py", line 65, in create_client
    cls = self._create_client_class(service_name, service_model)
  File "/home/user/.local/lib/python2.7/site-packages/botocore/client.py", line 90, in _create_client_class
    base_classes=bases)
  File "/home/user/.local/lib/python2.7/site-packages/botocore/hooks.py", line 227, in emit
    return self._emit(event_name, kwargs)
  File "/home/user/.local/lib/python2.7/site-packages/botocore/hooks.py", line 210, in _emit
    response = handler(**kwargs)
  File "/home/user/.local/lib/python2.7/site-packages/boto3/utils.py", line 61, in _handler
    module = import_module(module)
  File "/home/user/.local/lib/python2.7/site-packages/boto3/utils.py", line 52, in import_module
    __import__(name)
  File "/home/user/.local/lib/python2.7/site-packages/boto3/s3/inject.py", line 15, in <module>
    from boto3.s3.transfer import create_transfer_manager
  File "/home/user/.local/lib/python2.7/site-packages/boto3/s3/transfer.py", line 129, in <module>
    from s3transfer.manager import TransferConfig as S3TransferConfig
  File "/home/user/.local/lib/python2.7/site-packages/s3transfer/manager.py", line 21, in <module>
    from s3transfer.utils import get_callbacks
  File "/home/user/.local/lib/python2.7/site-packages/s3transfer/utils.py", line 27, in <module>
    from botocore.exceptions import ReadTimeoutError
ImportError: cannot import name ReadTimeoutError

**Comment by erinn** [comment]
Confirmed setting boto3==1.9.8 fixed the issue, I just picked a low version no reason that is the 'right' one. 1.8 didn't work.

**Comment by biblicabeebli** [comment]
Ah, in the cluster management subproject requirements file.

I think the best fix here is to remove botocore from requirements altogether, as it is a dependency of boto.  (Not sure why it was there to begin with...)

I just tested, installing only boto3 in a fresh python virtual environment installs, among other things, boto3-1.9.153 and botocore-1.12.153.

(boto3 is a library that Amazon Web Services maintains, but it is _generated_ from a specification, not developed by humans.  The most recent boto3 library is nearly always desirable.)

I've pushed an updated launch_requirements file to master.

Thank you for bug report!
